tests/test_deliverables.py

```diff
@@ -228,7 +231,7 @@ class TestMilestones(unittest.TestCase):
     @classmethod
     def setUp(cls):
         cls.testHelper = TestHelper(my_account)
-        cls.team_pk = my_account.teams_pk[0]['id']
+        cls.team_pk = TeamFactory()
```

Comment:
pass org_id ?

Reply:
TeamFactory does a request on 'v1/profiles/profile/'. Seems to get info on logged profile:
```
{
   ....
   selected_org": 9095,
  "selected_team": 8668,
   ....
  }
```